packages/twenty-front/src/modules/command-menu/components/CommandMenuContextChip.tsx

```diff
@@ -7,6 +7,7 @@ const StyledChip = styled.button<{
   withText: boolean;
   onClick?: () => void;
 }>`
+  all: unset;
```

Comment:
Wouldn't this unset the default font? I don't understand why we would do that?

Reply:
OK got it https://stackoverflow.com/questions/26140050/why-is-font-family-not-inherited-in-button-tags-automatically ; maybe best to set font-family: inherit ? Like we do in Textarea for example

Reply:
(or use theme.font.family)